ipython-profile.py

```diff
@@ -1,2 +1,3 @@
-from galaxy import get, put, get_galaxy_connection, _get_history_id
+from galaxy_ie_helpers import get, put
+from galaxy get_galaxy_connection, _get_history_id
```

Comment:
btw, the change doesn't fix it. I believe. nate mentioned not being able to import galaxy_ie_helpers.